net/uradvd/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uradvd
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/freifunk-gluon/uradvd.git
+PKG_SOURCE_DATE:=2021-08-14
+PKG_SOURCE_VERSION:=9b0da60e27c67305d251b10163e388191d566d7a
+PKG_MIRROR_HASH:=70dac4c156cd98baea92c9f1fb14692737b6c55c2aa0442c272ec9abe8fe06e4
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uradvd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Advertise an IPv6 prefix/route
+endef
+
+define Package/uradvd/description
+	Advertise an IPv6 prefix from the node
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+	$(CP) ./src/* $(PKG_BUILD_DIR)/
+endef
+
+define Build/Configure
+endef
+
+define Build/Compile
+	CFLAGS="$(TARGET_CFLAGS)" CPPFLAGS="$(TARGET_CPPFLAGS)" $(MAKE) -C $(PKG_BUILD_DIR) $(TARGET_CONFIGURE_OPTS)
+endef
+
+define Package/uradvd/install
+	$(CP) ./files/* $(1)/
```

Comment:
While we use it in Gluon a lot, I don't like `$(CP)` for the install step that much anymore. When installing only a few files, it is IMO advisable to do the `$(INSTALL_DIR)`, `$(INSTALL_BIN)` and `$(INSTALL_DATA)` manually, as this will ensure that no unexpected permission bits are set on files.

Reply:
I actually like `$(CP)`, as it decouples the static files to be installed and the Makefile.
Nevertheless, I have chnaged it.